### DISCUSSION TITLE
Heading tags is not recognised in my tiptap editor

### DISCUSSION THREAD

**Discussion by Nithur-M** [ueberdosis/tiptap#3430] [General Questions]
I am trying to toggle the heading on a text, but it is not working. Also, if I set the content with any heading tags, it's also not rendering the heading style. Why?

[edit]: the editor is not rendering blockquote also.

**Comment by juicerq** [comment]
You're probably using tailwind

**Reply by Nithur-M** [reply]
What is the workaround for that?

**Reply by juicerq** [reply]
I manually set the value for h1, h2... in a css file with 

```
#editor h1 {
  textsizehere...
}
```

this happens because tailwind automatically resets headings

**Reply by Nithur-M** [reply]
why not simply use tailwind/typography?
